src/platform/baytrail/include/platform/mem_map.h

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2016, Intel Corporation
```

Comment:
Use current your for new headers (also in other places).

Reply:
done